core-primitives/types/src/lib.rs

```diff
@@ -12,16 +12,6 @@ use sp_runtime::{
 use std::vec::Vec;
 
 use itp_storage::storage_entry::StorageEntry;
-pub use its_primitives::{
-	traits,
-	types::{
-		block,
-		block::{
-			BlockHash, BlockNumber as SidechainBlockNumber,
-			ShardIdentifier as SidechainShardIdentifier,
-		},
-	},
-};
```

Comment:
So this is the core change of this PR? along with the change in `sidechain/primitives/src/types/mod.rs` ?

Reply:
Yes. You spotted it correctly. But as many files were affected, I wanted to have it separate from the aura stuff.